### PR TITLE
Add AGPL License

### PR DESCRIPTION
**Pull request by plstcharles**
The project still does not have a license; Joumana pointed me towards this one which seems like a really good idea. In short, it's a modified GPL that also applies the "redistribution" clause to potential SaaS applications.

We need this merged in before switching to the centralized repo.

### CODE REVIEW

**Comment by plstcharles** [comment]
There's a background discussion underway on AGPL vs GPL, let's not merge this quite yet --- will update if necessary.

**Comment by plstcharles** [comment]
Will be merged on Covi-Canada once everything is fixed instead.

**Comment by plstcharles** [comment]
We got the OK for AGPL, this should be merged into master for the (deprecated) public repo landing page.

**Review by pg2455 — APPROVED** [review]

**Pull request merged**
